internal/tofu/transform_targets.go

```diff
@@ -115,14 +228,70 @@ func (t *TargetsTransformer) selectTargetedNodes(g *Graph, addrs []addrs.Targeta
 
 		if found > 0 {
 			// we found an output we can keep; add it, and all it's dependencies
-			targetedNodes.Add(v)
+			targetedOutputNodes.Add(v)
 			for _, d := range deps {
-				targetedNodes.Add(d)
+				targetedOutputNodes.Add(d)
 			}
 		}
 	}
 
-	return targetedNodes, nil
+	return targetedOutputNodes
+}
+
+func (t *TargetsTransformer) nodeIsExcluded(vertexAddr addrs.Targetable, excludes []addrs.Targetable) bool {
+	for _, excludeAddr := range excludes {
+		// The behaviour here is a bit different from targets.
```

Comment:
This comment assumes that the reader has knowledge of what TargetContains() does. Maybe it's best to move this comment into that method?

Reply:
09ae600ac387435db24b5d23c5ad66be430d89b8

Just removed the comment, as it makes more sense in the function under it.
That function also follows up with an explanation on the addresses used, so it should clarify that